### PR TITLE
fix(react-tinacms-github): added action.name as key prop in actions.m…

### PR DESCRIPTION
**Pull request by dangayle**
added action.name as key prop in actions.map method

fix https://github.com/tinacms/tinacms/issues/1557

### CODE REVIEW

File: packages/react-tinacms-github/src/components/ActionableModal.tsx [hunk 1]
```diff
@@ -50,8 +50,12 @@ export const ActionableModal = ({
             <p>{message}</p>
           </ModalBody>
           <ModalActions>
-            {actions.map(action => (
-              <TinaButton primary={action.primary} onClick={action.action}>
+            {actions.map((action) => (
+              <TinaButton
+                key="name"
```
Comment (dwalkr) [inline packages/react-tinacms-github/src/components/ActionableModal.tsx:55]: key should not be a constant https://reactjs.org/docs/lists-and-keys.html#keys

**Review by dwalkr — CHANGES_REQUESTED** [review]

**Comment by dangayle** [comment]
Oh god, what a noob mistake. I will fix.

**Comment by dwalkr** [comment]
@dangayle FYI we also need contributors to this project to sign the Contributor License Agreement. It looks the commits in this PR aren't tied to your GitHub user. In order for CLAassistant to approve this, all committers need to be tied to GitHub.

You can fix this by setting a user email for git that matches an email associated with your GitHub account: https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address#setting-your-commit-email-address-in-git

You will then need to amend + force push your current commit to replace the author.

Sorry for making this so complicated, but once you set this up it shouldn't be an issue in the future!

**Comment by dangayle** [comment]
Yeah, I did the CLA, but didn’t know I needed to amend the push. I’ll do
that. Thank you for your patience.

**Comment by dwalkr** [comment]
@dangayle I'm gonna close this PR as it's been open with no activity for a little while. You're welcome to reopen it if/when you're ready to come back to it!

**Pull request closed without merging**